--- modules/ngx_http_xquic_module/ngx_xquic_intercom.c
+++ modules/ngx_http_xquic_module/ngx_xquic_intercom.c
@@ -246,7 +246,7 @@
     c = ctx->connection;
 
     n = sendto(c->fd, packet, sizeof(ngx_xquic_recv_packet_t), 0,
-               &ctx->addr[worker_num], ctx->addrlen[worker_num]);
+               (struct sockaddr *)&ctx->addr[worker_num], ctx->addrlen[worker_num]);

     ngx_xquic_stat_send_cnt++;